sysboundary/sysboundarycondition.h

```diff
@@ -161,6 +161,14 @@ namespace SBC {
             const bool calculate_V_moments
         )=0;
 
+        virtual void setupL2OutflowAtRestart(
+            dccrg::Dccrg<SpatialCell, dccrg::Cartesian_Geometry>& mpiGrid
+        ) {
+            std::cerr << "ERROR: base class SysBoundaryCondition::setupL2OutflowAtRestart called!" << std::endl;
+        }
```

Comment:
I still continue to be in favour of actually *not* defining the base class implementations of virtual functions that we don't actually want called anyway. That way, it becomes a compile-time problem.

Does that work in this case, or is the code still too spaghetti to allow for it?

Reply:
We call loops over boundaries, as "always" – if not like this we need an implementation in every other derived class too, which I think we just removed recently?...